views/js/controller/Tools/assessmentActivity.js

```diff
@@ -196,7 +193,96 @@ define([
                         show: false
                     }
                 }
-            }).render();
+            })
+            .render();
+        } else {
+            activityGraph.refresh({
```

Comment:
after refreshing chart is broken:
![image](https://cloud.githubusercontent.com/assets/11025793/25841050/33e20df4-34a7-11e7-8986-0a3f6d849fd6.png)